.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: jammy
 sudo: required
 services: xvfb
 node_js:
-  - "20.15.1"
+  - "22.9.0"
```

Comment:
@denihs: Revert this since it causes the CI to fail, check out this comment for further information, https://github.com/meteor/meteor/pull/13252#discussion_r1731330223

We could try to fix it as part of this PR, but maybe is good to focus on the Node upgrade and adapt possible breaking changes if any, then focus more on the CI upgrade.

Reply:
Reverted. If possible, I think it would be better to spend some time to migrate Travis test to either CircleCI or GitHub to reduce the number of CI services.

Reply:
Yes, we are working towards using GitHub Actions for all checks. We have already moved some checks, but there are still many left. Still, it's better to spend time migrating the checks rather than fixing issues with services like Travis.